consumer_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafka
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/matryer/is"
+	"github.com/segmentio/kafka-go"
+)
+
+func TestMessagePosition(t *testing.T) {
```

Comment:
We already have the file `segment_reader_test.go`, we should either move this test there or move the others here (I would prefer the latter because it's customary to have `foo.go` and `foo_test.go`).

Reply:
Makes sense. This is a leftover from when newReader() was a function and not a method on the consumer.